haml/_input.html.haml

```diff
@@ -3,6 +3,8 @@
   .cads-form-content.cads-input-group
     %label.cads-form-label.cads-heading-small{id: "#{input['name']}-label", for: "#{input['name']}-input" }
       = input['label']
-    %p.cads-form__hint= input['hint']
-    %p.cads-form__error-message= input['error_message']
+    - if input['hint']
```

Comment:
The lack of if statements here was masking the bug. If you use these classes standalone and don't need a hint then the label would display inline.